kcbq-connector/src/main/java/com/wepay/kafka/connect/bigquery/BigQuerySinkTask.java

```diff
@@ -194,14 +200,24 @@ private PartitionedTableId getRecordTable(SinkRecord record) {
 
     PartitionedTableId.Builder builder = new PartitionedTableId.Builder(baseTableId);
     if (usePartitionDecorator) {
+
+      Table bigQueryTable = retrieveCachedTable(baseTableId);
+      TimePartitioning timePartitioning = null;
```

Comment:
Think we can initialize this to `DAY` partitioning here and save a few null checks down the road.

Reply:
Initialisation to DAY completed along with removal of the NULL checks